redis.cfg

```diff
@@ -33,12 +33,12 @@ redis-logfile = ${buildout:directory}/var/log/redis.log
 recipe = collective.recipe.scriptgen
 cmd = ${redis:location}/src/redis-server
 arguments =
-  ${buildout:directory}/redis.conf
+  ${buildout:bin-directory}/redis.conf
 
 [redis-conf]
 recipe = collective.recipe.template
-input = redis.conf.tpl
-output = redis.conf
+input = buildout-templates/redis.conf.tpl
+output = ${buildout:bin-directory}/redis.conf
```

Comment:
I don't think redis.conf should go in the bin directory. At the root of the project makes more sense.